apps/codeforafrica/src/components/ImageIcon/index.js

```diff
@@ -0,0 +1,32 @@
+import { ImageButton } from "@commons-ui/core";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+const ImageIcon = React.forwardRef(function Logo(props, ref) {
```

Comment:
```suggestion
const NextImageButton = React.forwardRef(function Logo(props, ref) {
```
Since it's based on a button, it's important for the name to indicate this is a button.